ui/admin/app/components/auth-methods/auth-method/actions/index.hbs

```diff
@@ -4,78 +4,64 @@
 }}
 
 {{#if (or @model.isOIDC @model.isLDAP)}}
-  <Rose::Dropdown
-    @text={{t (concat 'states.' @model.state)}}
-    @icon={{if
-      @model.isInactive
-      'flight-icons/svg/x-circle-16'
-      (if
-        @model.isPrivate
-        'flight-icons/svg/lock-16'
-        'flight-icons/svg/check-circle-16'
-      )
-    }}
-    @dropdownRight={{true}}
-    as |dropdown|
-  >
-    <dropdown.section @title={{t 'actions.change-state'}}>
-      <Rose::Form as |form|>
-        <form.radioGroup
-          @name='theme'
-          @selectedValue={{@model.state}}
-          @changed={{fn @changeState @model}}
-          as |radioGroup|
-        >
-          {{#each @model.options.state as |state|}}
-            <dropdown.item>
-              <radioGroup.radio
-                @label={{t (concat 'states.' state)}}
-                @value={{state}}
-              />
-            </dropdown.item>
-          {{/each}}
-        </form.radioGroup>
-      </Rose::Form>
-    </dropdown.section>
-  </Rose::Dropdown>
+  <Hds::Dropdown class='change-state' data-test-change-state as |dd|>
+    <dd.ToggleButton
+      @text={{t (concat 'states.' @model.state)}}
+      @color='secondary'
+      @icon={{if
+        @model.isInactive
+        'x-circle'
+        (if @model.isPrivate 'lock' 'check-circle')
+      }}
+    />
+
+    {{#each @model.options.state as |state|}}
+      <dd.Radio
+        checked={{eq @model.state state}}
+        @value={{state}}
+        {{on 'click' (fn @changeState @model state)}}
```

Comment:
i think for this radio component we should use `{{on 'change'}}` instead because a request is made when the user clicks a radio field that is already selected. However, if we use `change` a new request is not made when a user clicks a already selected field.

Reply:
you are right, thanks for the catch, updated!